packages/storybook/.storybook/preview.js

```diff
@@ -1,6 +1,6 @@
-import '@department-of-veterans-affairs/formation/dist/formation.min.css';
+// import '@department-of-veterans-affairs/formation/dist/formation.min.css';
 import './style.scss';
-import '@department-of-veterans-affairs/formation/dist/formation';
+// import '@department-of-veterans-affairs/formation/dist/formation';
```

Comment:
The mobile components aren't like web components with a shadow dom so we will have to deal with how global styles may bleed into elements. 

For example, when these Formation imports are active, the border disappears from the secondary mobile components.

![Screenshot 2025-02-18 at 4 59 30 PM](https://github.com/user-attachments/assets/32d7fe0d-2967-40d4-a9e6-47a32cc0bc47)